### PR TITLE
Fix 73 consolidate search

### PR DESCRIPTION
**Pull request by navdeepsinghkhalsa**

### CODE REVIEW

File: www/js/banidb/index.js [hunk 1]
```diff
@@ -13,6 +12,7 @@ module.exports = {
     // default source for ang search to GURU_GRANTH_SAHIB
     let angSearchSourceId = CONSTS.SOURCE_TYPES.GURU_GRANTH_SAHIB;
     const order = [];
+    let howManyRows = 20;
```
Comment (tsingh777) [inline www/js/banidb/index.js]: Any specific reason for 20?

File: www/js/banidb/index.js [hunk 2]
```diff
@@ -55,6 +55,7 @@ module.exports = {
       }
       case CONSTS.SEARCH_TYPES.ANG: // Ang
         searchCol = 'PageNo';
+        howManyRows = 1000;
```
Comment (tsingh777) [inline www/js/banidb/index.js]: is this the max for any ang?

File: www/js/banidb/index.js (hunk 1 above)
Comment (navdeepsinghkhalsa) [reply www/js/banidb/index.js]: Just to keep is snappy

File: www/js/banidb/index.js (hunk 2 above)
Comment (navdeepsinghkhalsa) [reply www/js/banidb/index.js]: It's just ridiculously high to make sure all cases are covered.

**Pull request merged**